src/components/ContentWrapper.jsx

```diff
@@ -40,20 +41,37 @@ const ContentWrapper = (props) => {
   const [isAuth, setIsAuth] = useState(false);
   const [collapsed, setCollapsed] = useState(false);
 
-  const { experimentId, experimentData, children } = props;
+  const { routeExperimentId, experimentData, children } = props;
   const router = useRouter();
   const route = router?.route || '';
   const navigateTo = useAppRouter();
 
-  const experiment = useSelector((state) => state?.experiments[experimentId]);
+  const experiment = useSelector((state) => state?.experiments[routeExperimentId]);
   const experimentName = experimentData?.experimentName || experiment?.name;
+  const currentExperimentRef = useRef(routeExperimentId);
+  const activeProjectUuid = useSelector((state) => state?.projects?.meta?.activeProjectUuid);
+  const activeProjectExperiment = useSelector((state) => (
+    state?.projects[activeProjectUuid]?.experiments[0]));
 
+  // if the current selected project experiment is processed
+  // and we are in data-management, we don't have experimentId
+
+  useEffect(() => {
+    if (routeExperimentId && currentExperimentRef.current !== routeExperimentId) {
+      currentExperimentRef.current = routeExperimentId;
+      return;
+    }
+    if (!routeExperimentId && currentExperimentRef.current !== activeProjectExperiment) {
+      currentExperimentRef.current = activeProjectExperiment;
+    }
+  }, [routeExperimentId, activeProjectExperiment]);
+
+  const currentExperiment = currentExperimentRef.current;
```

Comment:
it seems `currentExpreiment` contains the `id`. Can you rename this variable to `currentExprimentId`? I think you should also change `currentExprerimentRef` to `currentExperimentIdRef`